rrfs-test/validated_yamls/templates/obtype_config/adpsfc_airTemperature_181.yaml

```diff
@@ -55,9 +55,10 @@
            where:
            - variable: QualityMarker/airTemperature
              is_in: 4-15
+             value: is_not_valid
+             where operator: or
```

Comment:
This "where operator: or" should use same indentation as "where:"

Reply:
@delippi My earlier test somehow messed up. Actually we don't need that duplicate variable line.

Reply:
Oh my... I just added it all.
Yep, my script put the where operator at the wrong indentation. I had noticed that and fixed it.

Reply:
It looks cleaner without. I will make those changes tomorrow. Thanks!